src/display/tilemap.cpp

```diff
@@ -1233,10 +1231,9 @@ void Tilemap::update()
 		p->flashAlphaIdx = 0;
 
 	/* Animate autotiles */
-	if (!p->tiles.animated)
-		return;
-
-	++p->tiles.aniIdx;
+	// Perry: This is now handled in the ruby code by calling 
+	//		  TilemapUtils::updateAutotileAniIdx because the autotile animation
+	//        index is shared between all tilemaps.
```

Comment:
```suggestion
	// Perry: This is now handled in the ruby code by calling `TilemapUtils::updateAutotileAniIdx`
	// because the autotile animation index is shared between all tilemaps to avoid desync on map connection.
```